### ISSUE TITLE
How to build the source?

### ISSUE THREAD

**Issue by FriskTheFallenHuman** [Bforartists/Bforartists#3724]
Hi! :) i wanna know how i can build bforartist under manjaro? does this uses blender official build wiki? or uses a different build system?

**Comment by Draise14** [comment]
Yes, the build process would be the same as Blender, with the key change being the repository you have to download from Bforartists here on Github, but the rest is about the same. 

This blog post may have more information. 
https://blenderartists.org/t/building-blender-in-manjaro/1198314

If not, follow these instructions, and when cloning the repository, use this repository instead of Blenders.
https://wiki.blender.org/wiki/Building_Blender/Linux/Arch
